apiBase/apiRequest.js

```diff
@@ -20,9 +20,13 @@ export default (apiOptions, method, path, options={}) => {
     ? { ...defaultHeaders, Authorization: `Bearer ${token}` }
     : defaultHeaders;
   const _path = path.startsWith('/') ? path : `/${path}`
-  const _query = { ...query, app: `${appName}-${env}` };
   const endpoint = origin + _path;
 
+  let _query = { ...query, app: `${appName}-${env}` };
+  if (_method === 'get') {
+    _query = { ...query, raw_json: 1 };
```

Comment:
It looks like this will remove the `app` param for GET requests. Is that intentional?